pages/admin/content-manager/[entity_type_slug].js

```diff
@@ -161,7 +161,7 @@ export default function ContentManager({cache}) {
 
             <div className="d-flex flex-row px-0"> 
               <AppDropdown title={ state.columns.length + ' items selected'} items={state.columns} id='dropdown_general' isCheckbox={true}/>
-              <AppIconButton icon={<BsGearFill/>} /> 
+              {/* <AppIconButton icon={<BsGearFill/>} />  */}
```

Comment:
Can we delete it instead of commenting this one out?

Let's be brave in deleting code, as we can easily revert to it via Git.